modules/swagger-codegen/src/main/resources/Java/ApiClient.mustache

```diff
@@ -458,11 +458,13 @@ public class ApiClient {
    *   application/json
    *   application/json; charset=UTF8
    *   APPLICATION/JSON
+   *   application/vnd.company+json
    * @param mime MIME
    * @return True if MIME type is boolean
    */
   public boolean isJsonMime(String mime) {
-    return mime != null && mime.matches("(?i)application\\/json(;.*)?");
+    String jsonMime = "(?i)^(application/json|[^;/ \t]+/[^;/ \t]+[+]json)[ \t]*(;.*)?$";
```

Comment:
Yes, according to the grammar described here in [RFC-7230](https://tools.ietf.org/html/rfc7230#section-3.2).  You can see that `HTAB` corresponds to horizontal tab here in [RFC-5234](https://tools.ietf.org/html/rfc5234#appendix-B.1).